bin/wpc_init

```diff
@@ -121,7 +134,22 @@ set -e
 ## This code will be run during setup, INSIDE the container.
 ##
 
-wp core install --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=Site
-# wp theme activate ...
-# wp plugin activate ...
+##############
+# Config
+##############
+title=your_site_title
```

Comment:
@RobjS That sounds familiar. I'm going to be this is the source of the problem: https://github.com/dxw/wpc/blob/master/images/wordpress/Dockerfile#L13 And I think the solution will involve reading a man page and replacing one character on that line.